.travis.yml

```diff
@@ -2,12 +2,19 @@ language: scala
 jdk: oraclejdk7
 sudo: false
 before_install: umask 0022
+cache:
+  directories:
+   - $HOME/.scalding.dep.cache
 script:
   - "echo no op"
+addons:
+  apt:
+    packages:
+    - md5deep
 matrix:
   include:
 #BASE TESTS
```

Comment:
I think I can hack that in with a bash script that will validate this file refers to each target twice maybe -- other than some blacklists. Its not pretty but I think it should work? Thoughts?